src/app/logic/player.js

```diff
@@ -0,0 +1,44 @@
+export class Player {
+  constructor() {
+    this.correctAnswersScore = 0;
+    this.amountOfAnsweredQuestions = 0;
+    this.detailQuestionData = [];
+    this.endGameData = {};
+  }
+
+  registerAnswer(correctAnswer, playerAnswer, isanswerCorrect) {
```

Comment:
Argument "isanswerCorrect" could be written in camel case: "isAnswerCorrect"

Reply:
done! will be in new commit